cmd/kubeadm/app/cmd/upgrade/node.go

```diff
@@ -94,6 +109,56 @@ func NewCmdUpgradeNodeConfig() *cobra.Command {
 	return cmd
 }
 
+// NewCmdUpgradeControlPlane returns the cobra.Command for upgrading the controlplane instance on this node
+// Important; this command IMPORTANT. This command should be executed only after upgrading another control plane instance with `kubeadm upgrade apply`
```

Comment:
I don't think this comment adds much and it's missing a few words. the same text is already in the description, no need to add it here